src/dsmr/fields.h

```diff
@@ -300,9 +302,9 @@ namespace dsmr
     /* Meter Reading electricity delivered by client (Tariff 2) in 0,001 kWh */
     DEFINE_FIELD(energy_returned_tariff4, FixedValue, ObisId(1, 0, 2, 8, 4), FixedField, units::kWh, units::Wh);
     /*
- * Extra fields used for Luxembourg and Lithuania
+ * Extra fields used for Luxembourg, Lithuania and Austria
  */
-    DEFINE_FIELD(total_imported_energy, FixedValue, ObisId(1, 0, 3, 8, 0), FixedField, units::kvarh, units::kvarh);
+    DEFINE_FIELD(total_imported_energy, FixedValue, ObisId(1, 0, 3, 8, 0), FixedField, units::kvarh, units::varh);
```

Comment:
@geduxas 
I am no sure what you mean exactly..?

but shall not do that change the trick?

BR

Reply:
Oh its templates, hard to read, also i think i was worng also by implementing these lines of code.. 
Cloud you add dump of your meeter for those obis values?